### PR TITLE
Feature/39076 loadtest live pin gen

### PR DESCRIPTION
**Pull request by GuyHarwood**
- [x] beanshell script to support multi-node load tests with unique teachers across each node
- [x] configurable school count for dummy data injection
- [x] script to purge and rebuild local db with dummy data injection
- [x] try it out pin gen scenario
- [x] live pin gen scenario
- [x] live pin gen with check start and check submission

### CODE REVIEW

File: deploy/sql/config.js [hunk 1]
```diff
@@ -36,5 +36,8 @@ module.exports = {
     MaxRetryAttempts: parseInt(process.env.RETRY_MAX_ATTEMPTS, 10) || 3,
     InitialPauseMs: parseInt(process.env.RETRY_PAUSE_MS, 10) || 5000,
     PauseMultiplier: parseFloat(process.env.RETRY_PAUSE_MULTIPLIER) || 1.5
+  },
+  DummyData: {
```
Comment (jon-shipley) [inline deploy/sql/config.js]: I think I would prefer this to be in the main config as this is not strictly speaking SQL config.

File: load-test/check-submit-proxy/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (jon-shipley) [inline load-test/check-submit-proxy/.vscode/launch.json]: This vscode dir should probably not be checked in.

Comment (GuyHarwood) [reply load-test/check-submit-proxy/.vscode/launch.json]: its optional.  the vscode team do check it in, so they can share debugger config etc.
there is nothing in here that is user specific, it provides support for the azure functions tooling.

i'd be happy with `.idea` and `.vscode` folders, as long as they only include settings that are useful & non-invasive to other team members.

not fussed either way

Comment (jon-shipley) [reply load-test/check-submit-proxy/.vscode/launch.json]: I did caveat it with "probably".  If you intentionally checked it in then by all means leave it.

File: deploy/sql/config.js (hunk 1 above)
Comment (GuyHarwood) [reply deploy/sql/config.js]: it is in the main config file, for the sql app.

**Pull request merged**